src/lib/transaction.ts

```diff
@@ -29,14 +29,14 @@ const joinPath = (path: Path): string =>
   `${path.cosignerIndex}/${path.change}/${path.addressIndex}`
 
 export const send = async (
-  userToken: string, walletId: string, recipients: Recipient[], xprv?: string, password?: string
+  userToken: string, walletId: string, recipients: Recipient[], xprv?: string, passphrase?: string
```

Comment:
why not `password`?

Reply:
Because I wanted to make a distinction between user's password and passphrase to encrypt wallet